profile-dotemacs.el

```diff
@@ -72,13 +72,16 @@
 
 ;;; Code:
 
+(require 'cl)
 (require 'thingatpt)
 (require 'benchmark)
 
 ;; User variables
 
-(defvar profile-dotemacs-file "~/.emacs.d/init.el"
-  "File to be profiled.")
+(defconst profile-dotemacs-user-init-files '("~/.emacs"
+                                             "~/.emacs.el"
+                                             "~/.emacs.d/init.el")
```

Comment:
I think we should encourage best practice by putting `~/.emacs.d/init.el` at the beginning of the list.

Reply:
Sure. I presume the other repository used this order as at least according to the [Emacs manual](https://www.gnu.org/software/emacs/manual/html_node/emacs/Find-Init.html#Find-Init) that is the order Emacs looks for such files. I presume changing the order here won't require a change to this [line](https://github.com/doolio/profile-dotemacs/blob/30daa059abb397d79e853011a470f395df848c7c/profile-dotemacs.el#L115).

Reply:
You are right on both counts. I guess it would actually make more sense to match Emacs' own behavior, even if I don't think that behavior is very well-thought-out. (In my opinion, Emacs should check for `~/.emacs.d/init.el` first, since the other files are kind of outdated and sort-of-deprecated.)